src/utils/validation.ts

```diff
@@ -117,9 +117,9 @@ export function checkPoolSpaceAvailable(
 	sub: Submission,
 	overlapping: Reservation[]
 ) {
```

Comment:
previously, `startTs.indexOf(sub.endTime)` would return `-1` for `endTime=20:00` because `startTs` only included up to `19:30`.  `startEndTs` includes all possible start/end times.